services/common/src/components/reports/ReportGetStarted.tsx

```diff
@@ -35,7 +37,49 @@ interface ReportGetStartedProps {
   formButtons: ReactNode;
 }
 
-export const RenderPRRFields: FC<{ mineGuid: string }> = ({ mineGuid }) => {
+export const ReportInfoBox: FC<{ mineReportDefinition: IMineReportDefinition; verb: string }> = ({
+  mineReportDefinition,
+  verb,
+}) => {
+  return (
+    <div className="report-info-box">
+      {mineReportDefinition && (
+        <div>
+          <Typography.Title level={4} className="primary-colour">
```

Comment:
there was much extracting of functions, but adding primary-colour className was the only real change from the previous version,